app/views/home/reference_calculator.haml

```diff
@@ -12,7 +12,7 @@
         $('input[data-has-name-short]').each(function() {
           var bank_account = $(this).attr('data-bank-account') || default_bank_account;
           var name = $(this).attr('name');
-          var value = Math.ceil(parseFloat($(this).val()) * 100) / 100;
+          var value = Math.round(parseFloat($(this).val()) * 100) / 100;
```

Comment:
What's the rational for this change? I choose `ceil` to make sure, that people always get an amount which includes all of the required money. If 3 people they to pay 1/3 of 10 entering `3.333` will be rounded to `3.33` and results in `9.99`. So the foodcoop misses `0.01` otherwise it gets `0.02` plus (in case people do not enter the exact values anyway).

Reply:
As I already mentioned, with `ceil` some numbers with two decimals are wrongly ceiled to the next 0.01 step. Try `1.09`, `1.10`, `1.11`, `1.12` for example. The reason is that they are handled as `float` and not cut off at the last digit entered.
![refcalc](https://user-images.githubusercontent.com/22921479/108609633-7c8d6d80-73cf-11eb-9fb8-fb708fa6dcca.png)

> If 3 people they to pay 1/3 of 10 entering 3.333 

I don't understand that sentence, why should several people (an ordergroup consisting of 3 people?) enter a single fraction into the calculator? Or do you mean if someone wants to pay 10.00 in total, split evenly on 3 transaction types?
I don't see why this calculator should handle more than two decimals in the first place. The input field turns red as a warning if more than two decimals are entered.